201-2-vms-loadbalancer-natrules/azuredeploy.json

```diff
@@ -223,16 +160,16 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
+      "apiVersion": "2016-09-01", 
       "type": "Microsoft.Network/loadBalancers/inboundNatRules",
```

Comment:
There is no "inboundNatRules" child resource of "Microsoft.Network/loadBalancers". There's a property named "inboundNatRules". Did you mean to put these values there?